### DISCUSSION TITLE
How to delete a Deta Project?

### DISCUSSION THREAD

**Discussion by vivekkj123** [deta/cloud-docs#459] [Help]
Hi 👋🏼 ,
     

   I can't see a button / endpoint for deleting a Deta Project. Is there an option for that?

---
Thank you for this wonderful product 🔥

**Comment by jnsougata** [comment]
There's currently no way to delete any project. You may create another project! There's no limitation.

**Reply by el-garro** [reply]
Another question. The hability to delete is going to be added eventually or is a "feature"? I Have 2 projects I created just for testing before knowing that and it really triggers some weird OCD seein them there. Awesome plattform btw.
